### PR TITLE
Doc snippet for the module generator

### PR DESCRIPTION
**Pull request by Faryshta**

### CODE REVIEW

File: generators/module/default/module.php [hunk 1]
```diff
@@ -16,10 +16,19 @@
 
 namespace <?= $ns ?>;
 
+/**
+ * Class to initialize the module `<?= $generator->moduleID ?>`
```
Comment (klimov-paul) [inline generators/module/default/module.php]: 'Class to initialize the module' does not sounds appropriate.
Docs for the module class usually contains description of its purpose, like 'module for user private messages management' or something.

Comment (Faryshta) [inline generators/module/default/module.php]: what should i put there then?

**Comment by Faryshta** [comment]
any update on this? @klimov-paul @SilverFire ?

File: generators/module/default/controller.php [hunk 2]
```diff
@@ -13,8 +13,14 @@
 
 use yii\web\Controller;
 
+/**
+ * Default controller for the `<?= $generator->moduleID ?>` module
+ */
 class DefaultController extends Controller
 {
+    /**
+     * render the default view for the module
```
Comment (SilverFire) [inline generators/module/default/controller.php]: `Render` -> `Renders`

Comment (SilverFire) [inline generators/module/default/controller.php]: `default` -> `index`

File: generators/module/default/module.php (hunk 1 above)
Comment (SilverFire) [inline generators/module/default/module.php]: What about

`<?= $generator->moduleID ?> module definition class`

**Comment by Faryshta** [comment]
proposed changes applied

**Comment by SilverFire** [comment]
Merged, thank you!

**Pull request merged**